src/track/cue.h

```diff
@@ -64,8 +49,8 @@ class Cue : public QObject {
 
   private:
     explicit Cue(TrackId trackId);
-    Cue(int id, TrackId trackId, Cue::Type type, double position, double length,
-        int hotCue, QString label, PredefinedColorPointer color);
+    explicit Cue(TrackId trackId, int sampleRateHz, const mixxx::CueInfo& cueInfo);
```

Comment:
Use `mixxx::SampleRate sampleRate` for type safety. The unit _Hz_ is implicitly part of this wrapper type.

Reply:
Ok. Should I change the getters/setters for `Track` to this type, too?

Reply:
> Ok. Should I change the getters/setters for `Track` to this type, too?

That would include a bigger refactor which is out of scope for this PR.

Reply:
No, just leave `Track` as it is. This is all legacy code that has not been updated.